src/library_html5_webgl.js

```diff
@@ -439,9 +439,9 @@ var LibraryHtml5WebGL = {
   },
 
   emscripten_webgl_get_supported_extensions__proxy: 'sync_on_current_webgl_context_thread',
-  emscripten_webgl_get_supported_extensions__deps: ['$stringToNewUTF8'],
+  emscripten_webgl_get_supported_extensions__deps: ['$stringToNewUTF8', '$getEmscriptenSupportedExtensions'],
   emscripten_webgl_get_supported_extensions: () =>
-    stringToNewUTF8(GLctx.getSupportedExtensions().join(' ')),
```

Comment:
This function `emscripten_webgl_get_supported_extensions()` is called by native WebGL rendering engines (so groups 1 and 2 in https://github.com/emscripten-core/emscripten/issues/20798#issuecomment-1865200414), so this one should not get the allowlisted treatment.

Reply:
sgtm!

Reply:
Done